src/dbup-cli.tests/VariableSubstitutionTests.cs

```diff
@@ -82,5 +82,22 @@ public void LoadMigration_ShouldSubstituteVariablesToScript()
             Logger.Log.Should().Contain("print 'Var2Value'");
             Logger.Log.Should().Contain("print 'Var3 Value'");
         }
+
+        [TestMethod]
+        public void LoadMigration_ShouldNotSubstituteVariablesToScript()
+        {
+            var env = A.Fake<IEnvironment>();
+            A.CallTo(() => env.GetCurrentDirectory()).Returns(@"c:\test");
+            A.CallTo(() => env.FileExists(A<string>.Ignored)).ReturnsLazily(x => File.Exists(x.Arguments[0] as string));
+
+            var engine = new ToolEngine(env, Logger, (testConnectionFactory as IConnectionFactory).Some());
+
+            var result = engine.Run("upgrade", GetConfigPath("disable-vars.yml"));
+            result.Should().Be(0);
+
+            Logger.Log.Should().NotContain("print 'Var1Value'");
```

Comment:
Hmm, there should be no output since this is a test that uses my `disable-vars.yml` config file which has `disableVars` param set to true. By using that no logs should be rendered which is what I'm excepting in my test.

Reply:
Sorry, I wrote the comment without checking the test - but when I run it, the log shows e.g. ```Execute non query command: print '$Var1$'```, so I believe checking for the print and the preserved var should work.

Reply:
My test fails when I'm expecting the `print 'Var1Value'` line. Weird. Let me check again...

Reply:
@huesie Now I'm checking if there are vars that are not substitutioned in the logs. Can you take a look?

Reply:
New commit looks great @anisalibegic